### DISCUSSION TITLE
Cannot upgrade with quasar cli

### DISCUSSION THREAD

**Discussion by rNoz** [quasarframework/quasar#9828] [CLI - PWA mode] [answered]
I have quasar/cli installed globally, and a recent project (a week ago) that is v1.15.21 (quasar) and v2.2.10 (quasar/app).

If I try to upgrade to the latest version, it gives not upgradeable packages. What is wrong?

```
$ quasar inspect

 Dev mode.......... spa
 Pkg quasar........ v1.15.21
 Pkg @quasar/app... v2.2.10
 Debugging......... enabled

$ quasar help
  Running @quasar/cli v1.2.1
  Running @quasar/app v2.2.10

$ quasar upgrade

  Quasar CLI · Gathering information with yarn...

  Congrats! All Quasar packages are up to date.

$ quasar upgrade --install

  Quasar CLI · Gathering information with yarn...

  Congrats! All Quasar packages are up to date.
```

**Comment by MilosPaunovic** [comment, marked as answer]
Have you checked out official [upgrade guide](https://quasar.dev/start/upgrade-guide)?

**Reply by rNoz** [reply]
Sure, as you can see in my post. The `quasar upgrade` in my case differs completely with what I have seen in the Quasar videos.

More info, partial package.json:
``` 
 "dependencies": {
    "@quasar/extras": "^1.0.0",
    "axios": "^0.21.1",
    "core-js": "^3.6.5",
    "quasar": "^1.0.0",
    "vue-class-component": "^7.2.2",
    "vue-property-decorator": "^8.3.0"
  },
  "devDependencies": {
    "@quasar/app": "^2.0.0",
  ...
```

**Reply by metalsadman** [reply]
read all the upgrade guide, you're probably just reading this https://quasar.dev/start/upgrade-guide#with-quasar-cli.
read further down for the guide from quasar v1 to quasar v2. starts from https://quasar.dev/start/upgrade-guide#migrate-to-v2-from-v1.

**Comment by rstoenescu** [comment]
You are on latest Quasar v1. Nothing to upgrade to (as of writing these lines).
If you want to upgrade to Quasar v2, then take a look at the [upgrade guide](https://quasar.dev/start/upgrade-guide) for it.

**Reply by rNoz** [reply]
Exactly. Thanks to all of you @metalsadman @rstoenescu @MilosPaunovic
